### DISCUSSION TITLE
Source file import options

### DISCUSSION THREAD

**Discussion by rorydavidson** [WeblateOrg/weblate#5974] [Q&A]
We are hosting our own Weblate server and I'm trying to import a source for translation that would need to include other metadata that could be seen when users are translating. They could include thing like a clickable URL to where each item is used and other potential pieces of information that would be visible on each item to help the translation. 

I've tried a few different file formats, including CSV, but no luck. These could potentially be included as a comment or an explanation on each item, but I've not found a way to import either yet.

Anyone else tried the same thing or got any suggestions?

**Comment by nijel** [comment]
The only methods with replace whole file are replace and source, see https://docs.weblate.org/en/latest/user/files.html#import-methods for more info.
